src/cljs/nr/gameboard.cljs

```diff
@@ -1725,6 +1738,7 @@
                 [log-pane]
                 [log-typing]
                 [log-input]]]
+              (do (resize-card-zoom) nil)
```

Comment:
Is there a better way to do this? I'm not familiar with reagant.

Reply:
What is the purpose of this line?

Reply:
(resize-card-zoom) is to perform the resize on load. The (do .. nil) is to return a nil so reagant ignores it. Basically I'm trying to run some javascript inline with the html generation.

Reply:
yeah, okay. I feel like it's not my favorite way to write that, but I can't think of a better way. I can always clean this up later if needed. (FAMOUS LAST WORDS)